views/index.erb

```diff
@@ -5,27 +5,49 @@
   <%= erb :header %>
 
   <script type="text/javascript">
-    tours['showoff:activity'] = [
-      {
-        element: ".currentSlide .activityToggle",
-        intro: I18n.t('tour.activity.complete')
-      }
-    ];
-
-    tours['showoff:form'] = [
-      {
-        element: ".currentSlide input.save",
-        intro: I18n.t('tour.form.save')
-      }
-    ];
-
-    tours['showoff:menu'] = [
-      {
-        element: "#hamburger",
-        intro: I18n.t('tour.menu')
-      }
-    ];
-
+    $( document ).ready(function() {
```

Comment:
Because this is for the audience view specifically. Same with the code in `presenter.erb`